lib/staccato/v4/tracker.rb

```diff
@@ -0,0 +1,335 @@
+# frozen_string_literal: true
+
+require_relative 'adapter_defaults'
+require_relative 'event' # must load first, to dynamically define event methods
+
+module Staccato::V4
+  # The `Tracker` class has methods to create all `Hit` types
+  #   using the measurement_id, api_secret, and client_id.
+  #
+  # @author Tony Pitale
+  class Tracker
+    extend Staccato::V4::AdapterDefaults
+
```

Comment:
Oops.  I forgot to delete this line.  That's what I get for committing a quick fix from GitHub's web UI on my phone. 😉 

```suggestion
```